README.md

```diff
@@ -51,7 +51,12 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+// var pizza = {
+//  temperature: 70,
+//  blake = function () {
```

Comment:
Close. When defining methods in object, you need to stick with key-value pair notation. Your object should look something like this...

```js
var pizza = {
   temperature: 70,
   bake: function( ){
      this.temperature = 300;
   }
}
```